newstroke_hershey/convert.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+NewStroke/Hershey to FontoBene conversion script.
+"""
+import os
+from ctypes import *
+
+HEADER = """\
+# This font was automatically converted from StrokeFont to FontoBene.
+# - StrokeFont project:        http://vovanium.ru/sledy/newstroke/en
+# - FontoBene specifications:  https://github.com/fontobene/fontobene
+# - Converter script:          https://github.com/fontobene/fontobene-converters
+#
+# As the StrokeFont is released under the CC0-1.0 license, the converted
+# FontoBene font is released under the same license. CC0 licence text:
+# http://creativecommons.org/publicdomain/zero/1.0/
+
+[format]
+format = FontoBene
+format_version = 1.0
+
+[font]
+name = FontoStroke
+id = fontostroke
+version = 1.0
+author = StrokeFont Developers
+author = FontoBene Developers
+license = CC0-1.0
+letter_spacing = 1.8
+line_spacing = 16
+
+---
+"""
+
+REPLACEMENT_GLYPH = 'F^K[KFYFY[K['
+
+
+def format_number(num):
+    """
+    Properly format a floating point number according the fontobene specifications.
+    """
+    rounded = round(float(num), 2)
```

Comment:
Any particular reasons why rounding to 2 digits of precision?

Reply:
I wouldn't cut off too much since bad rounding *might* lead to rendering issues depending on zoom level and panning. 2 digits is probably fine.